hackathon_site/dashboard/frontend/src/components/general/SideSheetR/SideSheetR.js

```diff
@@ -0,0 +1,165 @@
+import React from "react";
+import Drawer from "@material-ui/core/Drawer";
+import Typography from "@material-ui/core/Typography";
+import Link from "@material-ui/core/Link";
+import Button from "@material-ui/core/Button";
+import MenuItem from "@material-ui/core/MenuItem";
+import FormControl from "@material-ui/core/FormControl";
+import Select from "@material-ui/core/Select";
+import LaunchIcon from "@material-ui/icons/Launch";
+import Grid from "@material-ui/core/Grid";
+
+import styles from "./SideSheetR.module.scss";
+import { ReactComponent as BackArrow } from "../../../assets/images/icons/BackArrow.svg";
+import { ReactComponent as CircleType } from "../../../assets/images/icons/redtype.svg";
+
+const createQuantityList = (number) => {
+    var entry = [];
+
+    for (var i = 0; i < number; i++) {
+        entry.push(i + 1);
+    }
+
+    return entry;
+};
+
+const CartSheet = ({ cart, detail }) => {
+    const [item, setItem] = React.useState("");
+
+    const handleChange = (value) => {
+        setItem(value);
+    };
+
+    if (cart) {
+        return (
+            <div className={styles.cart}>
+                <FormControl className={styles.form}>
+                    <Select value={item} role="selected">
```

Comment:
Set the default value of the input to 1 and give it a label of "Qty". The input is also not the style in the mockup. The examples on the MUI select page is the same